test/testDB.js

```diff
@@ -1,17 +1,18 @@
 const tape = require('tape');
 const { addUser, blockUser } = require('../src/model/queries/users');
-const { addRoom, deleteRoom } = require('./../src/model/queries/rooms');
+const { addRoom, deleteRoom, getRoom } = require('./../src/model/queries/rooms');
 const db_build = require('./../src/model/database/db_build');
 
+
 //  testing build database
 tape('Testing Insert a new user to database', (t) => {
```

Comment:
is it a test to insert new user, or test build databse

and please remove the commented code